model/application.go

```diff
@@ -54,6 +54,14 @@ type Import struct {
 	ImportSummaryID     uint `gorm:"index"`
 	Processed           bool
 	ImportTags          []ImportTag `gorm:"constraint:OnDelete:CASCADE"`
+	BinaryGroup         string
+	BinaryArtifact      string
+	BinaryVersion       string
+	BinaryPackaging     string
+	RepositoryURL       string
+	RepositoryBranch    string
+	RepositoryTag       string
```

Comment:
I think the `RepositoryTag` can be removed.

Reply:
Good point, thanks!